scripts/package_building_bisect/build_artifacts_bisect.sh

```diff
@@ -407,11 +485,15 @@ function main () {
     repository=$(get_source "$BASE_DIR" "$SOURCE_PATH")
 
     get_sources_git "$BASE_DIR" "$SOURCE_PATH" "$GIT_BRANCH" "$GIT_COMMIT_ID" "$repository"
-    destination_path="$(get_destination_path "$repository" "$DESTINATION_PATH" "$os_PACKAGE" "${GIT_COMMIT_ID:0:7}" "$BUILD_DATE" "$FOLDER_PREFIX")"
+    destination_path="$(get_destination_path2 "$repository" "$DESTINATION_PATH" "$os_PACKAGE" "$BUILD_DATE" "$FOLDER_PREFIX")"
     prepare_kernel_"$os_FAMILY" "$BASE_DIR" "$repository" "$PACKAGE_PREFIX" "$DEP_PATH"
 
     build_"${os_FAMILY}" "$BASE_DIR" "$repository" "$kernel" "$THREAD_NUMBER" "$destination_path" \
-        "$BUILD_DATE" "$KERNEL_VERSION"
+        "$BUILD_DATE"
+
+    if [[ "$os_FAMILY" == "debian" ]]; then
+        build_daemons "$BASE_DIR" "$repository" "$DEP_PATH" "$DEBIAN_OS_VERSION" "$PACKAGE_PREFIX"
```

Comment:
change build_daemons to build_daemons_$os_FAMILY and update the corresponding method definition

Reply:
like in the package build original's code